tests/features/solution/solution_overview.feature

```diff
@@ -105,42 +156,3 @@ Feature: Solutions Overview
 
     # Clean up the solution that was created manually.
     And I delete the "Colonies in Earth" solution
-
-  @terms
-  Scenario: Custom pages should not be visible on the overview page
-    Given the following solution:
```

Comment:
These two tests are about the solution homepage, not about the solution overview, so I moved them to `homepage.feature`.